episodes/severity-static.Rmd

```diff
@@ -572,9 +572,7 @@ Is this pattern similar to other outbreaks? We can use the data sets in this epi
 
 ::::::::::::::::::::::
 
-:::::::::::::::::::::: discussion
-
-### Checklist
+:::::::::::::::::::::: checklist
```

Comment:
As discussed today, this is done on purpose, to allow having foldable divs after.